HealthcareSystem/Backend/Service/UsersAndWorkingTime/DoctorService.cs

```diff
@@ -76,5 +76,10 @@ public Doctor ViewProfile(string jmbg)
         {
             return _doctorRepository.GetDoctorByJmbg(jmbg);
         }
+
+        public List<Doctor> ViewDoctorsBySpecialty(int specialtyId)
```

Comment:
It should be Get, not View.

Reply:
I know, but I followed convention from the Backend of Sims project that our company uses. I will change that in some of the future commits.